server/datastore/mysql/software_installers.go

```diff
@@ -156,6 +172,25 @@ func (ds *Datastore) getOrGenerateSoftwareInstallerTitleID(ctx context.Context,
 	return titleID, nil
 }
 
+func (ds *Datastore) addSoftwareTitleToMatchingSoftware(ctx context.Context, titleID uint, payload *fleet.UploadSoftwareInstallerPayload) error {
+	whereClause := "WHERE (s.name, s.source, s.browser) = (?, ?, '')"
```

Comment:
Is that fully equivalent to `s.name = ? AND s.source = ? AND s.browser = ''` for index-use and all? If so that's good to know, nice shorthand!

Reply:
my understanding: in this case it is because it's a composite index on the three values!